packages/core/src/types.ts

```diff
@@ -184,6 +184,7 @@ export type UpsetConfig = {
     histograms: Histogram[];
   };
   allSets: Column[];
+  selected: Bookmark;
```

Comment:
Worth mentioning that a selection is NOT a Bookmark, rather a bookmark is a selection. I'd suggest adding a type for selection (same as `Bookmark`) and then making the `Bookmark` type an extension of this. The functionality is the exact same, but I think it will make it easier to differentiate the two in the future.

Reply:
I meant to make that a `Row`, and thought I made that change; not sure where that got lost. Do you think that's fine or should I add a different `Selection` type for semantic purposes?

Reply:
I think Row is fine provided it holds the same content. Semantically it makes sense that a selection is a Row object